.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
   - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=4648da44fc1695d5306e9b4f05e96fa5a5cadb8f
```

Comment:
Remember to revert before merging? Or if this is going to be the new official `25-hotfix` centaur branch, can we label it as such?